### ISSUE TITLE
The Python errors are not detected within {pythonrepl}

### ISSUE THREAD

**Issue by fpantigny** [tndrle/PyLuaTeX#19]
When there is a Python error in a code within an environment `{python}`, `pyluatex` returns an error message. However, when the error is in an environment `{pythonrepl}`, there is a problem and we have an odd error message: `bad argument #2 to 'lpegmatch' (string expected, got nil)`. Of course, the problem does not occur when `pyluatex` is loaded with the key `ignoreerrros`.

```
\documentclass{article}
\usepackage[executable=python.exe]{pyluatex}

\begin{document}

\begin{pythonrepl}
x = 4 
y = z # will trigger a Python error
\end{pythonrepl}

\end{document}
```

**Comment by tndrle** [comment]
Thank you very much for opening an issue. It's a bug I introduced in v0.6.0.
It's fixed in v0.6.1 which is out now but needs a few days to be available in
CTAN, TeX Live, ...

**Comment by fpantigny** [comment]
Thanks.

 

F. Pantigny

 

De : tndrle ***@***.***> 
Envoyé : mercredi 15 février 2023 21:00
À : tndrle/PyLuaTeX ***@***.***>
Cc : fpantigny ***@***.***>; Author ***@***.***>
Objet : Re: [tndrle/PyLuaTeX] The Python errors are not detected within {pythonrepl} (Issue #19)

 

Thank you very much for opening an issue. It's a bug I introduced in v0.6.0.
It's fixed in v0.6.1 which is out now but needs a few days to be available in
CTAN, TeX Live, ...
